massa-protocol-worker-2/src/handlers/block_handler/messages.rs

```diff
@@ -44,6 +44,8 @@ pub enum BlockInfoReply {
 }
 
 #[derive(Debug)]
+//TODO: Fix this clippy warning
+#[allow(clippy::large_enum_variant)]
```

Comment:
should box the largest enum ?

Reply:
It's annoying to use a variant that is a Box in matches as you probably saw with the BlockMessage in tests. So for now I prefer ignore and if we find a good way in the futur remove the allow that's why I placed a todo